packages/pds/src/app-view/api/app/bsky/unspecced.ts

```diff
@@ -147,6 +147,14 @@ export default function (server: Server, ctx: AppContext) {
             .as('likeCount'),
         ])
 
+      if (query) {
+        inner = inner.where((qb) =>
+          qb
+            .where('feed_generator.displayName', 'ilike', `%${query}%`)
```

Comment:
heads up: when you're working the pds package, we need to support both sqlite & postgres

I believe `ilike` is postgres only so this won't work with the sqlite impl :/

Kyesely abstracts over a lot of this, but this one, you'll probably need to do an if/else on `db.dialect`



Couple thoughts:
- this route will move to the appview soon so probably isn't _that_ important that we don't support both
- this would make search fail when running the dev-env without postgres (which I think is the general workflow for the frontend team)
- should be quick to add in so i lean towards doing so